==== build_tools/mozc_version.py#6 build_tools/mozc_version.py ====
--- build_tools/mozc_version.py#6	2011-06-02 16:56:40.000000000 +0900
+++ build_tools/mozc_version.py	2011-07-20 12:48:44.647117000 +0900
@@ -36,8 +36,6 @@ def CalculateRevisionForPlatform(revisio
   """Returns the revision for the current platform."""
   if not revision:
     return revision
-  if target_platform == 'ChromeOS':
-    last_digit = '4'
   elif IsWindows():
     last_digit = '0'
   elif IsMac():
